src/main/webapp/app/shared/layout/header/header.tsx

```diff
@@ -45,12 +47,23 @@ export default class Header extends React.Component<IHeaderProps, IHeaderState>
     return (
       <div id="app-header">
         <LoadingBar className="loading-bar" />
-        <Navbar dark className="bg-primary">
-          <NavbarToggler aria-label="Menu" onClick={this.toggleMenu} />
+        <Navbar dark>
+          <Button outline color="primary" aria-label="Menu" onClick={this.toggleMenu}>
+            <FontAwesomeIcon icon="bars" />
+          </Button>
           <Brand />
           <Nav id="header-tabs" className="ml-auto">
             <LocaleMenu currentLocale={currentLocale} onClick={this.handleLocaleChange} />
-            <AccountMenu isAuthenticated={isAuthenticated} />
+            {isAuthenticated ? (
+              <div>avatar image</div>
```

Comment:
Can put a default placeholder Image 1st instead of using text

Reply:
fixed at 7cab542084fdc658a802282bb77e029be06f61a8